tests/probes/filehash58/test_probes_filehash58.sh

```diff
@@ -43,10 +43,65 @@ function test_probes_filehash58 {
     return $ret_val
 }
 
+
+# $1: The chroot directory
+function test_probes_filehash58_chroot {
+
+    probecheck "filehash58" || return 255
+    require "sha1sum" || return 255
+
+    local ret_val=0;
+    local DF="$srcdir/check_filehash_simple.xml"
+
+    absolute_probe_root=$(cd "$1" && pwd)
+
+    # oscap-chroot is not readily available during test run, so we use oscap + env var setting.
+    result_keyword=$(OSCAP_PROBE_ROOT="$absolute_probe_root" "$OSCAP" oval eval "$DF" | grep oval_test_has_hash | grep -o '\w*$')
+    
+    [ "$result_keyword" == "$2" ] && return 0
+    # vvv This is more a test error than a failiure or "warning" vvv
+    [ "$result_keyword" == "" ] && return 2
+    return 1
+}
+
+
+function test_probes_filehash58_chroot_pass {
+	local ret_val=0
+
+	mkdir -p pass
+	echo foo > pass/oval-test
+
+	test_probes_filehash58_chroot pass true
+	ret_val=$?
+	rm -rf pass
+
+	[ $ret_val -eq 0 ] && rm -f /tmp/$FUNCNAME.tmp
+	return $ret_val
+}
+
+
+function test_probes_filehash58_chroot_fail {
+	local ret_val=0
+
+	mkdir -p fail
+	echo bar > fail/oval-test
+
+	test_probes_filehash58_chroot fail false
+	ret_val=$?
+	rm -rf fail
+
+	[ $ret_val -eq 0 ] && rm -f /tmp/$FUNCNAME.tmp
```

Comment:
Where is this directory created?

Reply:
Another good catch, I was puzzled by the first function of this test suite, fixed by rebase.